internal/testtypes/remediation.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testtypes
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	// RemediationGroupVersion is group version used for remediation objects.
+	RemediationGroupVersion = schema.GroupVersion{Group: "remediation.external.io", Version: "v1alpha3"}
```

Comment:
I'm not sure about changing the group, does this risk to introduce errors due to missing RBAC rules for this group?
Also, should we move this to v1alpha4 for consistency

Reply:
I can move it to v1alpha4, was just moving back for minimal diff, but it doesn't matter. I'm not sure how this can introduce RBAC errors, afaik we're completely ignoring RBAC in test env and the tests are all green.

Reply:
I changed the group away from infrastructure to be sure we don't have any weird bugs which rely on it being in the same group.

Reply:
I changed the version to v1alpha4 and replaced references through the `RemediationGroupVersion.String()`.

Just let me know if you prefer the current group name or if I should change it back.

Reply:
Ok, thanks for the explanation